### ISSUE TITLE
[BUG] Cannot undeprecate

### ISSUE THREAD

**Issue by thimpat** [npm/cli#5597]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

I did a

```
npm deprecate mylib 3.6.0 "Please upgrade to latest"
```

which deprecated the whole package as I forgot the @ symbol between mylib and the version.

Now, if I try to undeprecate with:

```
npm deprecate mylib ""
```
or individually with
```
npm deprecate mylib@3.5.0 ""
```

I have got a "code EUSAGE" error.

I am using npm version 8.19.2

### Expected Behavior

Following the npm documentation, It should un-deprecate when passing an empty message.

### Steps To Reproduce

1. In this environment...
Windows 11
node v16.17.0
npm v8.19.2

2.

...

3. Run

```
npm deprecate mylib "Some deprecation message"
npm deprecate mylib ""
npm deprecate mylib@1.0.0 ""
```

5. See error...

```
npm ERR! code EUSAGE
npm ERR!
npm ERR! Deprecate a version of a package
npm ERR!
npm ERR! Usage:
npm ERR! npm deprecate <package-spec> <message>
npm ERR!
npm ERR! Options:
npm ERR! [--registry <registry>] [--otp <otp>]
npm ERR!
npm ERR! Run "npm help deprecate" for more info

npm ERR! A complete log of this run can be found in:


```

### Environment

- npm:
- Node.js:
- OS Name:
- System Model Name:
- npm config:
```ini
; copy and paste output from `npm config ls` here
```

**Comment by JOU-amjs** [comment]
Yes, I have the same problem, and I changed many versions, v8.12.1, v7.19.1 and the latest version, but the whole didn't work, which made me headache...

can anybody resolve it?

**Comment by paulmillr** [comment]
Same on v9.

**Comment by mbtools** [comment]
I solved this riddle! The issue is caused by PowerShell. Empty parameters like `""` are ignored by PowerShell on Windows. Therefore, `npm deprecate` will never get `""` as input. 

The issue is solved with PowerShell 7.3 or higher by changing how parameters are handled. Enter the following on the command line. Then `npm deprecate package ""` will work as expected

```powershell
$PSNativeCommandArgumentPassing="Standard"
```

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_preference_variables?view=powershell-7.4#psnativecommandargumentpassing
